### ISSUE TITLE
Unable to serve docs locally via `hugo serve`

### ISSUE THREAD

**Issue by jpmcb** [vmware-tanzu/community-edition#3687]
## Bug Report

I am unable to serve docs locally via `hugo serve`:

![docs-404](https://user-images.githubusercontent.com/23109390/159807596-341759a4-1a4c-496d-a4e6-4d52fd3e67cc.gif)

Git commit off main:
```
❯ git rev-parse HEAD
44f2bc40b7a4c67603d2bda4bf333060235225d6
```

Fails with 404 when attempting to access docs under the `docs/v0.10` route. If I go to _just_ the `docs/` route it lands me here:
 
![image](https://user-images.githubusercontent.com/23109390/159807836-b1c1c0e0-be53-4108-8e8e-9f29c714f139.png)

But that's not very helpful for local docs writing. If I hit the drop down for the versioned docs, it doesn't work.

## Expected Behavior

After the versioned docs change, I'd still expect to be able to write docs and serve them locally for viewing and iterating.

## Steps to Reproduce the Bug

Pull down `main` and run `hugo serve`

**Comment by jpmcb** [comment]
@jonasrosland - any idea what's going on?

**Comment by garrying** [comment]
I had to run [`load-docs.sh`](https://github.com/vmware-tanzu/community-edition/blob/main/docs/site/load-docs.sh) to get the versioned docs loaded locally. @stmcginnis maybe an update to readme is needed?

**Comment by joshrosso** [comment]
Moving this to `v0.13.0. so we don't lose site of this.

It's probably worth re-evaluating the whole process end-to-end.

**Comment by jpmcb** [comment]
Would it be possible to serve a subdomain to a different statically built site with something like [Docusaurs](https://docusaurus.io/)? That would _automatically_ give us versioned docs and other nice to have docs features

So, if I go to `docs.tanzucommunityedition.io`, I am served the static files built by Docusaurs instead of our own site

**Comment by kartiklunkad26** [comment]
>It's probably worth re-evaluating the whole process end-to-end.

This makes sense, but any kind of temporarily solution would be really helpful!

**Comment by jonasrosland** [comment]
`make preview-build` or `make production` should both work. The prior showing draft and future content (blog posts dated in the future as an example), and the latter is what Netlify uses.

**Comment by kartiklunkad26** [comment]
@jonasrosland could you share more explicit instructions? 

- Should I still run `hugo server --disableFastRender`?
- What folder should I run `make preview-build` from?

**Comment by kcoriordan** [comment]
fixed by #4319
